core/store.js

```diff
@@ -30,7 +30,11 @@ const initialState = {
         value: '',
     },
   },
-  blueprints : [],
+  blueprints: {
```

Comment:
There init state should be an empty array.
From blueprints reducer file core\reducers\blueprints.js, it initialized state to an empty array. they're should be the same or remove blueprints initialization  from store.js.
```javascript
const blueprints = (state = [], action) => {
  switch (action.type) {
    case CREATING_BLUEPRINT_SUCCEEDED:
      return [
        ...state.filter(blueprint => blueprint.present.id !== action.payload.blueprint.id), {
          past: [],
          present: Object.assign({}, action.payload.blueprint,  {
            localPendingChanges: [],
            workspacePendingChanges: {addedChanges: [], deletedChanges: []},
            images: [],
          }),
          future: [],
        }
      ];
```